src/js/media-display-gesture-receiver.js

```diff
@@ -0,0 +1,134 @@
+    }
+
+    this.nativeEl.appendChild(slotTemplate.content.cloneNode(true));
+
+    shadow.appendChild(buttonHTML);
+
+    this._pointerType = undefined;
+    const pointerDownHandler = (e) => {
+      // Since not all browsers have updated to be spec compliant, where 'click' events should be PointerEvents,
+      // we can use use 'pointerdown' to reliably determine the pointer type. (CJP).
+      this._pointerType = e.pointerType;
+    };
+
+    this.addEventListener('pointerdown', pointerDownHandler);
+
+    this.addEventListener('click', (event) => {
+      const { pointerType = this._pointerType } = event;
+      // NOTE: While there are cases where we may have a stale this._pointerType,
+      // we're guaranteed that the most recent this._pointerType will correspond
+      // to the current click event definitionally. As such, this clearing is technically
+      // unnecessary (CJP)
+      this._pointerType = undefined;
+
+      // NOTE: Longer term, we'll likely want to delay this to support double click/double tap (CJP)
+      if (pointerType === PointerTypes.TOUCH) {
+        this.handleTap(event);
+        return;
+      } else if (pointerType === PointerTypes.MOUSE) {
+        this.handleMouseClick(event);
+        return;
+      }
+    });
+  }
+
+  attributeChangedCallback(attrName, oldValue, newValue) {
+    if (attrName === MediaUIAttributes.MEDIA_CONTROLLER) {
+      if (oldValue) {
+        const mediaControllerEl = document.getElementById(oldValue);
+        mediaControllerEl?.unassociateElement?.(this);
+      }
+      if (newValue) {
+        const mediaControllerEl = document.getElementById(newValue);
+        mediaControllerEl?.associateElement?.(this);
+      }
+    }
+  }
+
+  connectedCallback() {
+    this.setAttribute('tabindex', -1);
+    this.setAttribute('aria-hidden', true);
+
+    const mediaControllerId = this.getAttribute(
+      MediaUIAttributes.MEDIA_CONTROLLER
+    );
+    if (mediaControllerId) {
+      const mediaControllerEl = document.getElementById(mediaControllerId);
+      mediaControllerEl?.associateElement?.(this);
+    }
+  }
+
+  disconnectedCallback() {
+    const mediaControllerSelector = this.getAttribute(
+      MediaUIAttributes.MEDIA_CONTROLLER
+    );
+    if (mediaControllerSelector) {
+      const mediaControllerEl = document.getElementById(mediaControllerId);
+      mediaControllerEl?.unassociateElement?.(this);
+    }
+  }
+
+  // NOTE: Currently "baking in" actions + attrs until we come up with
+  // a more robust architecture (CJP)
+  handleTap(_evt) {}
+
+  handleMouseClick(_evt) {
+    const eventName =
+      this.getAttribute(MediaUIAttributes.MEDIA_PAUSED) != null
+        ? MediaUIEvents.MEDIA_PLAY_REQUEST
+        : MediaUIEvents.MEDIA_PAUSE_REQUEST;
+    this.dispatchEvent(
+      new window.CustomEvent(eventName, { composed: true, bubbles: true })
+    );
+  }
+}
+
+defineCustomElement(
+  'media-display-gesture-receiver',
```

Comment:
any chance this element name could be made shorter? 

I know the display here is the video display but it could potentially be read as this element is displaying a gesture receiver which is a bit strange :)
> Use `-display` when the primary use is displaying a state or data detail. e.g. `media-duration-display`.

maybe just `media-gesture-receiver`? or should it be more specific?

---

I was looking up some synonyms and stuff for receiver and `media-gesture-control` came up. 
it puts it more in the category of one of the media controls.
Receiver for me feels more like it implies a sender and a time where something is transmitting but might be overthinking it 😅

Reply:
This was based on an "off the dome" name proposal from Heff. Since we're not really advertising it just yet, I'm cool with renaming it while it's still tucked away as an internal if you feel strongly. `media-gesture-receiver` and `media-gesture-control` both sound like reasonable names to me. Preference?